catalyst_voices/packages/internal/catalyst_voices_services/lib/src/user/dummy_user_service.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:catalyst_cardano_serialization/catalyst_cardano_serialization.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+
+/// Creates dummy users and accounts.
+abstract interface class DummyUserService {
+  static const dummyKeychainId = 'TestUserKeychainID';
+  static const dummyUnlockFactor = PasswordLockFactor('Test1234');
+  static final dummySeedPhrase = SeedPhrase.fromMnemonic(
+    'few loyal swift champion rug peace dinosaur '
+    'erase bacon tone install universe',
+  );
+
+  factory DummyUserService() {
+    return const DummyUserServiceImpl();
+  }
+
+  User getDummyUser({String keychainId = dummyKeychainId});
+
+  Account getDummyAccount({String keychainId = dummyKeychainId});
```

Comment:
it its just suppose to create dummy user when it would be just a factory. Possible `UserService` should depend on it

Reply:
UserService already depends on it. I'll rename to DummyUserFactory and rename these methods:
- `getDummyUser` -> `buildDummyUser`
- `getDummyAccount` -> `buildDummyAccount`

because the factory is more likely to `build` than to `get` things